src/App.tsx

```diff
@@ -57,6 +57,13 @@ const AfterUnsubscribe = lazy(() =>
   import('screens/Subscriptions/AfterUnsubscribe'),
 );
 
+const metricExplainerHashes = [
+  '#icu-capacity-used',
+  '#daily-new-cases',
```

Comment:
This was taken from what we currently have in the code. I think `daily-new-cases` hash is dead though. We can probably get rid of this and have it redirect to the top of the page.

Reply:
I think it was replaced by `weekly-new-cases` (if you click the "About this data" link under the weekly new cases chart and Learn more, that's where you go), so we should maybe use that instead.